sample/src/progress.html

```diff
@@ -0,0 +1,17 @@
+<template>
+  <require from='./progress.css'></require>
+    <ai-dialog>
+        <ai-dialog-body>
+            <div class='blockPageDialog'>
```

Comment:
Can you fix the tabs so that they are all two spaces?  (README.md also has mixed spacing)